lib/install/templates/static.rb

```diff
@@ -0,0 +1,35 @@
+# Installs modules to enable static assets compiling and digesting with webpack
+
+require "json"
+require "webpacker/package_json"
+
+copy_file "#{File.expand_path("..", __dir__)}/config/static.js",
+          "config/webpack/static.js"
+
+static_config = <<-EOS
+const { webpacker } = require('../../package.json').config
+
+if (webpacker.static) {
+  const staticConfig = require('./static.js')
+  sharedConfig.config = merge(sharedConfig.config, staticConfig)
+}
+EOS
+
+insert_into_file "config/webpack/development.js",
+                  static_config,
+                  after: "const sharedConfig = require('./shared.js')\n"
+
+insert_into_file "config/webpack/production.js",
+                  static_config,
+                  after: "const sharedConfig = require('./shared.js')\n"
+
+run "./bin/yarn add extract-text-webpack-plugin node-sass file-loader " \
+    "sass-loader css-loader style-loader "
+
+package_json = Webpacker::PackageJson.new().config
```

Comment:
what if the user already modified package json?

Maybe webpacker should merge? not replace?